tdnf.spec.in

```diff
@@ -0,0 +1,242 @@
+#
+# Copyright (C) 2019-2020 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the GNU General Public License v2 (the "License");
+# you may not use this file except in compliance with the License. The terms
+# of the License are located in the COPYING file of this distribution.
+#
+# tdnf spec file
+#
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+
+%define _tdnfpluginsdir %{_libdir}/%{name}-plugins
+
+Summary:        dnf/yum equivalent using C libs
+Name:           @PROJECT_NAME@
+Version:        @PROJECT_VERSION@
+Release:        1%{?dist}
+Vendor:         VMware, Inc.
+Distribution:   Photon
+License:        LGPLv2.1,GPLv2
+URL:            https://github.com/vmware/tdnf
+Group:          Applications/RPM
+
+Requires:       rpm-libs
+Requires:       curl
+Requires:       %{name}-cli-libs = %{version}-%{release}
+Requires:       libsolv
+
+BuildRequires:  popt-devel
+BuildRequires:  rpm-devel
+BuildRequires:  openssl-devel
+BuildRequires:  libsolv-devel
+BuildRequires:  curl-devel
+BuildRequires:  libmetalink-devel
+
+#plugin repogpgcheck
+BuildRequires:  gpgme-devel
+BuildRequires:  cmake
+BuildRequires:  python3-devel
+BuildRequires:  systemd
+
+%if %{with_check}
+BuildRequires:  createrepo_c
+BuildRequires:  glib
+BuildRequires:  libxml2
+%endif
+
+Obsoletes:      yum
+Provides:       yum
+
+Source0:        %{name}-%{version}.tar.gz
+
+%description
+%{name} is a yum/dnf equivalent which uses libsolv and libcurl
+
+%package    devel
+Summary:    A Library providing C API for %{name}
+Group:      Development/Libraries
+Requires:   %{name} = %{version}-%{release}
+Requires:   libsolv-devel
+
+%description devel
+Development files for %{name}
+
+%package	cli-libs
+Summary:	Library providing cli libs for %{name} like clients
+Group:		Development/Libraries
+
+%description cli-libs
+Library providing cli libs for %{name} like clients.
+
+%package	plugin-repogpgcheck
+Summary:	%{name} plugin providing gpg verification for repository metadata
+Group:		Development/Libraries
+Requires:   gpgme
+
+%description plugin-repogpgcheck
+%{name} plugin providign gpg verification for repository metadata
+
+%package	python
+Summary:	python bindings for %{name}
+Group:		Development/Libraries
+Requires:   python3
+
+%description python
+python bindings for %{name}
+
+%package automatic
+Summary:        %{name} - automated upgrades
+Requires:       %{name} = %{version}-%{release}
+%{?systemd_requires}
+
+%description automatic
+Systemd units that can periodically download package upgrades and apply them.
+
+%prep
+%autosetup
+
+%build
+mkdir -p build && cd build
+cmake \
+-DCMAKE_BUILD_TYPE=Debug \
+-DCMAKE_INSTALL_PREFIX=%{_prefix} \
+-DCMAKE_INSTALL_LIBDIR:PATH=lib \
+..
+make %{?_smp_mflags} && make python
+
+%check
+cd build && make %{?_smp_mflags} check
+
+%install
+cd build && make DESTDIR=%{buildroot} install
```

Comment:
Can you remove the installation of services as part of make install target? We can manually copy all services to %{_unitdir} in spec file.